cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -118,6 +120,16 @@ private void checkPotentialPortConflicts() {
     }
   }
 
+  // checks that kafka topic provided is a valid kafka topic
```

Comment:
We don't put function docs with //. Will be better if change it to /* for consistency or just get rid of it as its a private method and the method name is self descriptive.